rust-escrow/conditional-escrow/src/lib.rs

```diff
@@ -199,6 +203,28 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_get_deposits() {
+        let mut context = setup_context();
+
+        testing_env!(context
+            .signer_account_id(bob())
+            .attached_deposit(ATTACHED_DEPOSIT)
+            .build());
+
+        let expires_at = add_expires_at_nanos(100);
+
+        let mut contract = setup_contract(expires_at, MIN_FUNDING_AMOUNT);
+
+        contract.deposit();
+
+        assert_eq!(
+            contract.get_deposits(),
+            vec![(bob(),ATTACHED_DEPOSIT)],
+            "Only Bob must have deposited"
+        );
```

Comment:
Let's add another depositor, to actually assert this. Otherwise, indeed, `bob` is the only depositor.